.github/workflows/check-learning-path-links.yml

```diff
@@ -0,0 +1,78 @@
+name: 'Check Learning Path Links'
+on:
+  schedule: # Run once a week
+    - cron: '0 0 * * 1'
```

Comment:
I'd be in favor of this running every other week (or once a month). As it stands every week is a bit of noise for a manual review PR since there will always be changes everytime the HEAD commit is updated.